src/osmf/lib/HDSPlayer.as

```diff
@@ -156,9 +156,9 @@ package
      * @param {string} channelBranch It can be info, debug, warn, error or log.
      * @returns {boolean} True or false indicating success
      */
-    private function SendToDebugger(value:String, referrer:String = null, channelBranch:String = "info"):Boolean
+    private function SendToDebugger(value:String, referrer:String = null, channelBranch:String = "log"):Boolean
     {
-      var channel:String = "console." + channelBranch;
+      var channel:String = "OO." + channelBranch;
```

Comment:
since we only have OO.log i would recommend just not using the channelBranch here except to switch between OO and console.* (in case you want to raise an error on all builds you can do console.error).
Anyways doesn't matter for this release but would prevent bugs in the future